Add app-misc/idemia-awp package

ykshatroff opened the pull request:
... to enable working with new IDEMIA id cards

This idemia awp***.deb package contains some things I don't know the purpose of, I let them stay there fwiw. Also I'd be glad to correct anything I might have done wrong, as this is my first portage. As for the thing itself, it proved to work for me. (Got an ID card quite recently - didn't work without this package)

ykshatroff committed:
Add app-misc/idemia-awp package to enable working with new IDEMIA id cards

ykshatroff commented on app-misc/idemia-awp/metadata.xml:
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person"><email>urmet.saar@gmail.com</email></maintainer>
```
Not sure about who'd be the maintainer. Feel free to disagree :)

urmet commented:
Ugh. I hoped that I could delay having to deal with this junk(open-eid/linux-installer#37) until I get a new card myself.

Just two comments on first glance:
1. inheriting `unpacker` eclass would probably remove the need for a `src_unpack` function
2. `MY_PV` is unused and can be removed

ykshatroff committed:
Use unpacker eclass and remove unused var

ykshatroff commented:
Thanks for the feedback! Yep saw that issue but just wanted to have it working here :)
Removed `MY_PV`, added unpacker - thanks, googling didn't help a lot about .debs handling.

jaakristioja commented:
I suggest the ebuild be called app-misc/idemia-awp-bin instead.

ykshatroff committed:
Rename ebuild to idemia-awp-bin

ykshatroff commented:
Thanks, indeed it's more appropriate, renamed.
